### ISSUE TITLE
YHS: write helm chart(s) for installing/running Yunikorn History Server

### ISSUE THREAD

**Issue by richscott** [G-Research/yunikorn-history-server#19]
Write one (or more if necessary, for different configurations) helm charts for installing and running YHS in a Kubernetes cluster. See https://helm.sh/ for more information.

**Comment by sudiptob2** [comment]
Hi I would like to work on this issue. I have some previous experience with helm.

**Comment by sudiptob2** [comment]
Hi,
I need some clarification regarding the requirements. We can handle the database in a couple of ways: we can deploy the database in the cluster, or we can use Helm values for an external database. We might even be able to do both.

What do you think? I'm looking for some feedback.

**Comment by richscott** [comment]
@sudiptob2  That's a good question - I do not know. @dave-gantenbein Will the SPARK users running Yunikorn be using their own Postgres database?

**Comment by sudiptob2** [comment]
Hi,
I have started working on creating a Helm chart for `yhs`. After analyzing the project, it seems we need the following resources:
1. Deployment
2. ConfigMap
3. Secrets
4. Service (maybe)

I would appreciate any suggestions on whether these resources are sufficient. Additionally, I am uncertain if we should use a Service to expose the `yhs` web API. If we do create a Service, what type should it be? I noticed that Yunikorn Core deploys a `LoadBalancer` service for their web API.

Some relevant mentions: @richscott @dejanzele @dave-gantenbein
